test/elli_tests.erl

```diff
@@ -3,56 +3,89 @@
 -include("elli.hrl").
 
 -define(I2B(I), list_to_binary(integer_to_list(I))).
+-define(I2L(I), integer_to_list(I)).
 -define(README, "README.md").
+-define(VTB(T1,T2, LB, UB),
+        erlang:convert_time_unit(
+          get_timing_value(T2) -
+              get_timing_value(T1),
+          native,
+          micro_seconds) > LB andalso
+        erlang:convert_time_unit(
+          get_timing_value(T2) -
+              get_timing_value(T1),
+          native,
+          micro_seconds) < UB).
 
```

Comment:
Could we abstract away the `erlang:convert_time_unit(get_timing_Value(T2) - get_timing_value(T1), native, micro_seconds)`?